### PR TITLE
make production logs less noisy

### PR DESCRIPTION
**Pull request by butlerx**

### CODE REVIEW

File: service.js [hunk 1]
```diff
@@ -103,8 +114,8 @@ require('./migrate-psql-db.js')(function (err) {
             }
           }
           if (args.q.fields$) {
-            args.q.fields$.forEach(function (field, index) {
-              args.q.fields$[index] = '\"' + escape(field) + '\"';
+            args.q.fields$.forEach(function(field, index) {
+              args.q.fields$[index] = '"' + escape(field) + '"';
```
Comment (Wardormeur) [inline service.js]: Why is backslash removed ?

Comment (butlerx) [reply service.js]: my mistake, used the wrong linter

File: service.js [hunk 2]
```diff
@@ -39,26 +41,31 @@ seneca.options.sanitizeTextArea = {
...
-    console.error(util.inspect(err.stack));
-    console.trace();
+  if (err !== undefined && err.stack !== undefined) {
+    var error = {
+      date: new Date().toString(),
+      msg: 'FATAL: UncaughtException, please report: ' + util.inspect(err.stack),
+      err: util.inspect(err)
+    };
+    console.error(JSON.stringify(error));
+    process.exit(1);
```
Comment (Wardormeur) [inline service.js]: if the error has no stack, the error is swallowed ?

File: Dockerfile [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
 MAINTAINER butlerx <butlerx@notthe.cloud>
+ENV NODE_ENV=production
 ARG DEP_VERSION=latest
-RUN apk add --update git build-base python postgresql-client
-RUN mkdir -p /usr/src/app
+RUN apk add --update git build-base python postgresql-client &&\
+   mkdir -p /usr/src/app
 WORKDIR /usr/src/app
-ADD . /usr/src/app
+COPY . /usr/src/app
```
Comment (Wardormeur) [inline Dockerfile]: If you copy, changes won't be detected ?

Comment (butlerx) [reply Dockerfile]: no they will be

File: service.js (hunk 2 above)
Comment (butlerx) [reply service.js]: thats how its always been. but fair point

File: Dockerfile [hunk 4]
```diff
@@ -1,13 +1,14 @@
...
 RUN npm install && \
-    npm install cp-translations@$DEP_VERSION && \
+    npm install cp-translations@"$DEP_VERSION" && \
```
Comment (Wardormeur) [inline Dockerfile]: Ar eyou sure those double quotes won't be interepreted during the npm install ?

Comment (butlerx) [reply Dockerfile]: nope its just shell script docker treats everything in a run command as sh

**Pull request merged**